README.md

```diff
@@ -4,20 +4,22 @@
   </a>
 
   <p align="center">
-    InstantSearch.js is a JavaScript library for building performant and instant search experiences with <a href="https://www.algolia.com/?utm_source=instantsearch.js&utm_campaign=repository">Algolia</a>.
+    InstantSearch.js is a JavaScript library for building performant and instant search experiences in vanilla JS, React, Vue and Angular with <a href="https://www.algolia.com/?utm_source=instantsearch.js&utm_campaign=repository">Algolia</a>.
   </p>
 </p>
 
 ---
 
-[![Version][version-svg]][package-url]
 [![License][license-image]][license-url]
 [![Build Status][ci-svg]][ci-url]
-[![Pull reminders][pull-reminders-svg]][pull-reminders-url]
 
-InstantSearch.js is a vanilla JavaScript library that lets you create an instant-search result experience using [Algolia][algolia-website]’s search API. It is part of the InstantSearch family:
+[InstantSearch.js][instantsearch-docs] is a JavaScript library that lets you create an instant-search result experience using [Algolia][algolia-website]’s search API.
 
-**InstantSearch.js** | [React InstantSearch][react-instantsearch-github] | [Vue InstantSearch][vue-instantsearch-github] | [Angular InstantSearch][instantsearch-angular-github] | [React InstantSearch Native][react-instantsearch-github] | [InstantSearch Android][instantsearch-android-github] | [InstantSearch iOS][instantsearch-ios-github]
+There are multiple wrappers for popular frameworks, such as [React InstantSearch][react-instantsearch-docs], [Vue InstantSearch][vue-instantsearch-docs], and [Angular InstantSearch][angular-instantsearch-docs].
+
+It is part of the InstantSearch family which is declined for different platforms:
+
+**InstantSearch.js** | [Angular InstantSearch][instantsearch-angular-github] | [InstantSearch Android][instantsearch-android-github] | [InstantSearch iOS][instantsearch-ios-github]
```

Comment:
I can understand that adding the ".js" suffix on the library name makes sense when next to the Android / iOS versions, but I'd still suggest removing it for consistency, as it refers to the multi-flavor monorepo and could create confusion with the vanilla flavor of InstantSearch. Wdyt? 

```suggestion
**InstantSearch** | [Angular InstantSearch][instantsearch-angular-github] | [InstantSearch Android][instantsearch-android-github] | [InstantSearch iOS][instantsearch-ios-github]
```

Reply:
otherwise what we could too is:
```suggestion
**InstantSearch.js** | **React InstantSearch**| **React InstantSearch Hooks** | **Vue InstantSearch** | [Angular InstantSearch][instantsearch-angular-github] | [InstantSearch Android][instantsearch-android-github] | [InstantSearch iOS][instantsearch-ios-github]
```

Reply:
I settled for the first, might be confusing to have multiple labels without links before the `Packages` part.